templates/Dockerfile.template

```diff
@@ -6,12 +8,22 @@ FROM {{ .From }}
 # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
 RUN groupadd -r mongodb && useradd -r -g mongodb mongodb
 
+{{- if $mongo4 }}
```

Comment:
I think this is more about debian9, not about mongo4. Please change this condition to be related to base image.

Reply:
done